client/src/pages/Login/Login.tsx

```diff
@@ -39,11 +47,28 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={12} md={12} lg={12} elevation={6} component={Paper} square>
         <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
+          <AppBar position="static" color="primary" elevation={0} className={classes.appBar}>
+            <Toolbar className={classes.toolbar}>
+              <img src={LogoImage} alt="logo" className={classes.logo}/>
+              {/* <Typography variant="h2" noWrap className={classes.toolbarTitle}>
+            LovingSitter.
+              </Typography> */}
+
```

Comment:
If we don't need this code here, we shouldn't be pushing up commented code into main.